### ISSUE TITLE
TSCH keepalive interval too big for CC26xx/CC13xx nodes

### ISSUE THREAD

**Issue by atiselsts** [contiki-ng/contiki-ng#657]
The default keepalive interval is 12 seconds. In `tsch/tsch-conf.h`:

```
/* Time to desynch assuming a drift of 40 PPM (80 PPM between two nodes) and guard time of +/-1ms: 12.5s. */
#define TSCH_KEEPALIVE_TIMEOUT (12 * CLOCK_SECOND)

```

However, the RX_WAIT for CC13xx/CC26xx platforms is set to 1800 microseconds in total (max 900 microsecond sync error). I have nodes with 75 ppm drift, for which the 12 second interval is too long. Either
1) The first keepalive should be sent sooner after the association to let the adaptive sync kick in (however, it must be sent *at least* 4 seconds after the assoc, otherwise the adaptive sync will not use it)
2) Or the default period reduced to 10 seconds or less.

I would prefer 1 - it does seem adding yet another special-case, but is justified... also, similar to OpenWSN.

**Comment by simonduq** [comment]
Yes, I see nothing wrong with reducing `TSCH_KEEPALIVE_TIMEOUT` on this platform. Note that `TSCH_MAX_KEEPALIVE_TIMEOUT` also comes into play.
